recipes/controller.rb

```diff
@@ -26,7 +26,7 @@
 include_recipe 'memcached'
 include_recipe 'osl-openstack::identity'
 include_recipe 'osl-openstack::image'
-include_recipe 'osl-openstack::network'
+include_recipe 'osl-openstack::network' unless node['osl-openstack']['network_node']
```

Comment:
@knightsamar the attribute means there is a separate network node so it shouldn't include this. If there is a better name for the attribute, please let me know.